R/as-data-frame.R

```diff
@@ -73,3 +102,96 @@ as.data.frame.CrunchDataFrame <- function (x, row.names = NULL, optional = FALSE
     names(out) <- names(x)
     return(structure(out, class="data.frame", row.names=c(NA, -nrow(x))))
 }
+
+
+#' Merge a CrunchDataFrame
+#' 
+#' Merging a CrunchDataFrame with a local dataframe is experiemental and might 
+#' result in unexpected results. One known issue is that using `merge` on a 
+#' CrunchDataFrame will change the both the CrunchDataFrame used as input as 
+#' well as create a new CrunchDataFrame. 
+#' 
+#' `merge`ing a CrunchDataFrame with a local dataframe is useful in situations 
+#' where you have new information in your local R session that you want to 
+#' connect with Crunch data. For example, this is especially useful for making
+#' plots with crunch and non-crunch data. It produces a hybrid CrunchDataFrame
+#' that has the local data attached to it, but like normal CrunchDataFrames 
+#' it is still judicious about downloading data from the server only when it 
+#' is needed.
+#' 
+#' @param x a CrunchDataFrame
+#' @param y a standard data.frame
+#' @param by.x name of the variable to match
+#' @param by.y name of the variable to match
```

Comment:
Doesn't merge have a "by" argument for when by.x and by.y are the same? cf. https://github.com/Crunch-io/rcrunch/blob/master/R/join-dataset.R#L62